core/src/scene/sceneLoader.cpp

```diff
@@ -1566,6 +1566,7 @@ void SceneLoader::parseStyleParams(Node params, const std::shared_ptr<Scene>& sc
         }
         default:
             LOGW("Style parameter %s must be a scalar, sequence, or map.", key.c_str());
+            out.emplace_back(StyleParam::getKey(key));
```

Comment:
This should probably go into a `case NodeType::Null`, otherwise `Undefined` nodes would get the same treatment. Also we won't need to print that warning for `Null` nodes now.

Reply:
I forgot about `Undefined` nodes. 👍